test/jasmine/tests/scattergl_test.js

```diff
@@ -583,6 +584,48 @@ describe('end-to-end scattergl tests', function() {
         .catch(failTest)
         .then(done);
     });
+
+    it('@gl should reset the sanp to length after react and not to TOO_MANY_POINTS constant', function(done) {
+        function fig(num) {
+            var x = [];
+            var y = [];
+            for(var i = 0; i < num; i++) {
+                x.push(Math.random());
```

Comment:
Random numbers are dangerous in tests. If you really need this to look like random numbers, you can use `Lib.pseudoRandom()` (preceded by `Lib.seedPseudoRandom()`) as a repeatable alternative.

Reply:
@alexcjohnson thanks for the review.
Good call! Addressed for `scattergl_test` and other jasmine tests in https://github.com/plotly/plotly.js/pull/4323/commits/902b08491d4480fa341650dcc365b230885a0afa.

Reply:
Beautiful, thank you for extending that to the rest of the test suite!